components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -19,9 +19,13 @@
   "web_accessible_resources": [
   ],
   "background": {
-    "scripts": ["out/brave_extension_background.bundle.js"]
+    "scripts": ["out/brave_extension_background.bundle.js"],
+    "persistent": true
   },
   "externally_connectable": {
+    "ids": [
+      "*"
+    ],
```

Comment:
If we don't specify this, no extensions/apps will be able to communicate with the Brave extension per https://developer.chrome.com/extensions/manifest/externally_connectable